pkg/operation/botanist/controlplane/kubecontrollermanager/logging.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kubecontrollermanager
+
+import (
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+)
+
+const (
+	loggingParserName = "kubeControllerManagerParser"
+	loggingParser     = `[PARSER]
+    Name        ` + loggingParserName + `
+    Format      regex
+    Regex       ^(?<severity>\w)(?<time>\d{4} [^\s]*)\s+(?<pid>\d+)\s+(?<source>[^ \]]+)\] (?<log>.*)$
+    Time_Key    time
+    Time_Format %m%d %H:%M:%S.%L
+`
+	loggingFilter = `[FILTER]
+    Name                parser
+    Match               kubernetes.*` + v1beta1constants.DeploymentNameKubeControllerManager + `*` + v1beta1constants.DeploymentNameKubeControllerManager + `*
```

Comment:
Please define a constant for the `containerName`

```suggestion
    Match               kubernetes.*` + v1beta1constants.DeploymentNameKubeControllerManager + `*` + containerName + `*
```